src/client/components/UsersListComponent/UsersListStyle.css

```diff
@@ -1,33 +1,20 @@
-.container {
-  width: 90%;
-  height: 50%;
-  margin: auto;
-}
-.wrapper {
-  display: inline-flex;
-  flex-direction: row;
+.user-list {
+  display: flex;
+  flex-direction: column;
   align-items: center;
-  text-align: center;
   width: 80%;
+  margin: 2% auto;
 }
-.user-list {
+
+.user-item {
```

Comment:
You should scope this with `.user-list .user-item`. It's a good practice to do this always.

Reply:
I try read more why it is good practice use parent class and child class, but dont find. I think only because two class is more important than one class, if we have difference styling for the same element, styling with 2 css class will win. Mind you this, when you write about good practice? or why? https://css-tricks.com/a-use-case-for-a-parent-selector/